tests/chia_log/handlers/test_wallet_added_coin_handler.py

```diff
@@ -40,7 +44,12 @@ def testFloatPrecision(self):
         with open(self.example_logs_path / "small_values.txt", encoding="UTF-8") as f:
             logs = f.readlines()
 
-        events = self.handler.handle("".join(logs))
+        default_config = self.handler_config
+        no_filter_config = copy.deepcopy(default_config)
+        no_filter_config["min_mojos_amount"] = 0
+        no_filter_handler = WalletAddedCoinHandler(no_filter_config)
```

Comment:
I think you can do `self.handler_config[“min_mojos_amount”] = 0` to directly override without copies. And then add a `self.handler_config.clear()` cleanup hook before or after each test.